registry/repositories.go

```diff
@@ -232,16 +278,28 @@ func streamHarborProjectsPage(ctx context.Context, registry *Registry, c chan st
 			continue
 		}
 
+		var harborProjRepo string = ctx.Value("harborProjRepo").(string)
+
+		if strings.Contains(harborProjRepo, "v2.0") {
+			ctx = context.WithValue(ctx, "harborProjRepo", (harborProjRepo + project.Name + "/repositories"))
+		} else {
+			ctx = context.WithValue(ctx, "harborProjRepo", (harborProjRepo + string(project.ID)))
```

Comment:
I'm not sure that `string(project.ID)` does what you want it to. Did you mean to use `fmt.Sprint`?